lib/periodic_jobs.rb

```diff
@@ -19,9 +19,9 @@
   mgr.register('45 0 * * *', 'AppealsApi::NoticeOfDisagreementCleanUpWeekOldPii')
   # Remove PII of NoticeOfDisagreements that have 1) reached one of the 'completed' statuses and 2) are a week old
   mgr.register('45 0 * * *', 'AppealsApi::SupplementalClaimCleanUpPii')
-  # Ensures that appeal evidence received "late" (after the appeal has reached "success") is submitted to Central Mail
-  mgr.register('30 * * * *', 'AppealsApi::EvidenceSubmissionBackup')
   # Remove PII of SupplementalClaims that have 1) reached one of the 'completed' statuses and 2) are a week old
+  mgr.register('30 * * * *', 'AppealsApi::EvidenceSubmissionBackup')
+  # Ensures that appeal evidence received "late" (after the appeal has reached "success") is submitted to Central Mail
```

Comment:
Descriptions for jobs are _under_ the job registrations in this file, which may be why these got swapped accidentally. I think having descriptions above is more standard, but I didn't make any change to that here.